gitWorkFlow.mdown

```diff
@@ -28,5 +28,9 @@ WHEN READY TO MAKE YOUR PULL REQUEST
 - **A** assigns some other person / people (**B**) to review it
 - **B** makes comments, suggests changes
 - **A** whines about it, but makes the changes
-- Eventually, **B** approves and hits Merge Pull Request in github
-- The whole team should now do `git checkout master` then `git pull origin master`
```

Comment:
This change was more of a chore but I snuck it in here.  Super bad practice, but wanted to have the instructions for local delete available.  This file is never executed.